components/powersensor/powersensor.go

```diff
@@ -19,6 +19,42 @@ func init() {
 		RPCServiceDesc:              &pb.PowerSensorService_ServiceDesc,
 		RPCClient:                   NewClientFromConn,
 	})
+
+	registerCollector("Voltage", func(ctx context.Context, ps PowerSensor) (interface{}, error) {
```

Comment:
IS there a place to test that the collector is doing what we expect it to do? @agreenb

Reply:
I can't think of where to put a software test, but if you spin up a server with this fake sensor and add the datamanger, the data tab should collect the fake sensor's data. Verify this behaviour for now.

Reply:
yep i've verified it using the fake powersensor, was able to get the correct data synced back to app.viam

Reply:
We've been testing each collector with fake components and ensuring that the data is synced as expected to [app.viam](https://app.viam.com/data/view?view=sensors), so this is correct. Would ask that you test every new method and ensure that the data is formed correctly in the Sensors Tab.

Reply:
Olivia how do you feel about returning random floats to get a more live test from each method in the fake sensor? @agreenb would that be sufficient for functionality and verifying that the data is fully formed for each time stamp?

Reply:
Yep it would! Even the current PR format with arbitrary floats for each method is sufficient, if you ensure that they're all being captured/synced and showing up.

Reply:
just confirmed everything is being synced using the fake sensor :)